index.js

```diff
@@ -21,23 +21,24 @@ class RunTask {
         this.runOne(taskItem, data, eachDone);
       }, done);
     }
-    const fn = this.tasks[task];
+    let fn = this.tasks[task];
+    if (this.options.bind) {
+      if (typeof fn === 'function') {
+        fn = fn.bind({ blah: '123' });
+      }
+    }
     if (!fn) {
       return done(new Error(`${task} does not exist`));
     }
-    if (typeof fn.execute === 'function') {
-      onStart(task);
-      return fn.execute(data, (err, result) => {
-        onFinish(task);
-        return done(err, result);
-      });
-    }
+    fn = fn.execute ? fn.execute : fn;
```

Comment:
remove the check for fn.execute?  don't we need to be able to run it whether it's a runkit class or just a function?